kafka-ui-react-app/src/components/common/Search/Search.tsx

```diff
@@ -10,14 +12,28 @@ interface SearchProps {
   value?: string;
 }
 
+const IconButtonWrapper = styled.span.attrs(() => ({
+  role: 'button',
+  tabIndex: '0',
+}))`
+  height: 16px !important;
+  display: inline-block;
+  &:hover {
+    cursor: pointer;
+  }
+`;
 const Search: React.FC<SearchProps> = ({
   placeholder = 'Search',
   disabled = false,
   value,
   onChange,
 }) => {
   const [searchParams, setSearchParams] = useSearchParams();
+  const ref = React.createRef<HTMLInputElement>();
```

Comment:
why we are using `createRef` instead of `useRef` ?

Reply:
yeh i will replace with useRef it's more effectively